src/extensions/CaptionsExtension/CaptionsExtension.ts

```diff
@@ -1,40 +1,40 @@
 import { Instance } from '@src/Instance';
 import { Module } from '@src/Module';
 import { HTML5Player } from '@src/player/HTML5Player/HTML5Player';
-import { CaptionsChangeEventData, Events } from '@src/types';
+import { CaptionsChangeEventData, Events, Caption } from '@src/types';
 
 export class CaptionsExtension extends Module {
   public name: string = 'CaptionsExtension';
 
-  private tracks: HTMLTrackElement[];
-
   constructor(instance: Instance) {
     super(instance);
 
-    this.tracks = instance.config.captions.map(caption => {
```

Comment:
It's kind of useless to save the tracks when they can be gotten from `mediaElement.textTracks`, don't you think?